web/packages/dmt-core/src/hooks/useDocument.tsx

```diff
@@ -18,6 +26,7 @@ export function useDocument<T>(
   const [document, setDocument] = useState<T | null>(null)
   const [isLoading, setLoading] = useState<boolean>(true)
   const [error, setError] = useState<AxiosError<any> | null>(null)
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore